truss/cli/cli.py

```diff
@@ -1001,24 +1006,27 @@ def view_training(
     if not remote:
         remote = remote_cli.inquire_remote_name()
 
-    if job_id and not project_id:
```

Comment:
I think this whole function could be another good candidate to move to truss/cli/train, to declutter this file